packages/relay/src/Environment.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 
-import { GRAPHQL_URL } from 'react-native-dotenv';
+// import { GRAPHQL_URL } from 'react-native-dotenv';
 import { createEnvironment, createNetworkFetcher } from '@mrtnzlml/relay';
 
 export default createEnvironment({
-  fetcherFn: createNetworkFetcher(GRAPHQL_URL),
+  fetcherFn: createNetworkFetcher('http://localhost:4000'),
```

Comment:
some problem with `.env`?

Reply:
Yeah, I was not supposed to commit that. I will remove it